doc/man1/openssl.pod

```diff
@@ -491,6 +491,35 @@ Provides a terse summary of all options.
 
 =back
 
+=head2 Key, Certificate and CRL Options
+
+Keys, Certificates and CRLs are commonly given as file paths or as
+URIs.  Depending on the URI scheme, this may require using an engine
+as well, see the B<-engine> option for the subcommands where you can
+specify this sort of object.
+
+For input file paths, the format is automatically determined, so old
+format specifications like B<-inform PEM> are generally no longer
+necessary, and are ignored and considered deprecated even though still
+accepted.
+
+Keys that are offered by an engines are an exception, they can still
+be given like this:
+
+ -engine eng -key something -keyform ENGINE
+
+The exact same specification can also be given like this:
+
+ -key engine:eng:something
+
+This is a temporary measure until engines have learned to provide an
```

Comment:
I somehow suspect this won't be temporary and this wording will still be here in 10 years time. Perhaps reword?